web/js/layers/active.js

```diff
@@ -5,7 +5,7 @@ import util from '../util/util';
 import wvui from '../ui/ui';
 import layersInfo from './info';
 import layersOptions from './options';
-import palettesLegend from '../palettes/legend';
+import {palettesLegend} from '../palettes/legend';
```

Comment:
Just curious, why is this change being made? Don't both versions mean the same thing?

Reply:
Otherwise I get `Uncaught TypeError: (0 , _legend2.default) is not a function`. To be perfectly fair, I am not entirely sure why it is only working this way. There seems to be quite a few gotchas related to import and export naming which I need to review further to understand what the issue is in this particular case.
https://stackoverflow.com/questions/36795819/when-should-i-use-curly-braces-for-es6-import/36796281

Reply:
Ah, I understand now. If you're importing something by name, rather than the default, the `{}` are required.